101-XSS-Attack-Prevention/azuredeploy.json

```diff
@@ -610,7 +605,7 @@
             "value": "[parameters('sqlServerPassword')]"
           },
           "bacpacuri": {
-            "value": "https://templatebotstorage.blob.core.windows.net/bacpacs/contosoclinic.bacpac"
+            "value": "https://azbotstorage.blob.core.windows.net/bacpacs/contosoclinic.bacpac"
```

Comment:
move this to a parameter and use GEN-BACPAC-URI

Reply:
@bmoore-msft Extracted the bacpac URL as a param & set a GEN value